src/Classes/ViewHelpers/DinoViewHelper.php

```diff
@@ -75,4 +75,37 @@ public static function displaySingleDinoDetails(DinoEntity $dino): string {
             return $result;
     }
 
+    public static function displaySequentialDino(string $singleDinoDetail, int $id, int $dinoCount) : string
+    {
+        $prev = intval($id) > 1 ?  intval($id) - 1 : 1;
+        $next = intval($id) < intval($dinoCount) ? intval($id) + 1 : intval($dinoCount);
+
+        $render = '';
+
+        if($id < 0 || $id > $dinoCount){
+            return 'you are out of range';
```

Comment:
This should probably be wrapped in an html tag (e.g. <p>). There is a class called 'noticeMessage' which would probably take care of the styling for this.

Will need to update unit tests.